src/server/service/kv.rs

```diff
@@ -1200,7 +1201,10 @@ fn future_get<E: Engine, L: LockManager>(
         } else {
             match v {
                 Ok((val, statistics, perf_statistics_delta)) => {
-                    let scan_detail_v2 = resp.mut_exec_details_v2().mut_scan_detail_v2();
+                    let exec_detail_v2 = resp.mut_exec_details_v2();
+                    exec_detail_v2.mut_time_detail().set_kv_read_wall_time_ms(used_time as i64);
+                    let scan_detail_v2 = exec_detail_v2.mut_scan_detail_v2();
+                    scan_detail_v2.set_read_bytes(statistics.data.flow_stats.read_bytes as u64);
```

Comment:
How about put this logic into `statistics.write_scan_detail`.

Reply:
Note that: not only `data` cf is read.